content/_data/upgrades/2-222-1.adoc

```diff
@@ -0,0 +1,93 @@
+==== Old builds may be deleted by global build discarders
+
+Jenkins will by default periodically run build discarders configured in projects, even when no builds are currently running.
+**This may delete old builds of projects that have been configured with a more aggressive build discarder configuration since the last build was run.**
+
+We recommend reviewing project configurations for aggressive build discarder settings before upgrading Jenkins if this change in behavior could result in unexpected data loss.
+
+NOTE: We recommend creating backups of your Jenkins configuration and data regularly.
+
+
+
+==== Always enabled CSRF Protection
```

Comment:
link to the ticket is missing

Reply:
No ticket (and I don't want comments on pull requests from people who have no business commenting on code).